R/internals.R

```diff
@@ -39,31 +40,38 @@ spf <- function(...) stop(sprintf(...), call. = FALSE)
   reslist <- httr::content(req, "parsed")
 
   if (length(reslist) == 0) {
-    stop("Zero records match your filter. Nothing to return.\n",
+    warning("Zero records match your filter. Nothing to return.\n",
          call. = FALSE)
+    invisible(NULL)
```

Comment:
If you really want to return `NULL`, you need an explicit `return()` here now.

But do you really want to return `NULL`? From a type-stability POV, maybe it should have the usual form but `realist` has length 0.